.github/actions/deploy-release-v1/README.md

```diff
@@ -0,0 +1,43 @@
+# deploy-release-v1
```

Comment:
I think the name is a bit misleading, we aren't deploying anything here. We're moving released issues and creating a sync PR, maybe:

```suggestion
# move-released-issues-and-create-sync-pr-v1
```

Thoughts @maksym-shynkarenko @straker?

Reply:
I agree. This name was from the task so I thought it was confirmed. Will rename it